crates/egui/src/widgets/plot/mod.rs

```diff
@@ -583,6 +565,107 @@ impl Plot {
         self
     }
 
+    /// Set the x axis label of the main x-axis
+    ///
+    /// If no x-axis has been specified so far a new one will be created.
+    pub fn x_axis_label(mut self, label: impl Into<WidgetText>) -> Self {
```

Comment:
It would be nice to add an example of using these new functions to the doctest of `Plot` (perhaps as a new example)!